wiki/pc-modding.md

```diff
@@ -73,6 +73,9 @@ An all-in-one tool that lets you easily manage BeatSaber versions, maps, mods, a
 
 ### BeatSaberModManager
 
+:::warning NOTICE  
+BeatSaberMOdManager does not work at this moment. Please use an older version or temporary switch to one of the other options on this page.
```

Comment:
Older versions aren't recommended as they just won't work on 1.37.2 and above at all, they will just fail to load mods every single time. As for the .old.exe, that happens when they update the game, and this wasn't the issue with me after I tried loading 0.0.6 multiple times. Would probably word it then as "BeatSaberModManager may not load for some users, so be aware of this".